src/test/java/uk/gov/ida/rp/testrp/authentication/TestRpSessionFactoryTest.java

```diff
@@ -87,7 +87,6 @@ public void before() throws Exception {
 
         when(resourceContext.getResource(ContainerRequestContext.class)).thenReturn(containerRequestContext);
         when(containerRequestContext.getUriInfo()).thenReturn(uriInfo);
-        when(containerRequestContext.getUriInfo().getRequestUri()).thenReturn(new URI("http://uri"));
```

Comment:
Why does this line need to be removed? Otherwise all looks good and runs locally without issues 👍🏽

Reply:
We get a Mockito fails with a message about unnecessary mocking if that line is retained.